### PR TITLE
feat: update header and footer toolbars

### PR DESCRIPTION
**Pull request by n05la3**

### CODE REVIEW

File: docs/src/components/landing-page/MainLayoutHeader.vue [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+              :padding="`xs ${$q.screen.name}`"
+              class="text-weight-bold text-size-16"
+              flat
+          >
+            <q-menu v-if="navItem.subMenu" class="shadow-bottom-small">
+              <nav-dropdown-menu :nav-items="navItem.subMenu"/>
+            </q-menu>
+          </q-btn>
+        </div>
+        <q-form v-if="searchField" autocapitalize="off" autocomplete="off" spellcheck="false" class="search-form position-relative">
```
Comment (n05la3) [inline docs/src/components/landing-page/MainLayoutHeader.vue]: Change searchField to isSearchFieldActive

File: docs/src/layouts/MainLayout.vue [hunk 2]
```diff
@@ -93,28 +20,28 @@
             v-if="showFooterToolbar(footerIndex)"
             :label="label"
             :to="to"
-            color="lp-light"
+            color="white-54"
```
Comment (n05la3) [inline docs/src/layouts/MainLayout.vue]: This color and the the white background on which it is placed causes text to be invisible

File: docs/src/components/landing-page/MainLayoutHeader.vue [hunk 3]
```diff
@@ -0,0 +1,269 @@
+<template>
+  <q-header class="bg-lp-dark font-monserrat">
+    <q-toolbar :class="$q.screen.xs? 'shadow-bottom-small':''" class="primary-toolbar q-pl-lg q-pr-md justify-between items-stretch">
+      <q-btn v-if="$q.screen.xs" flat @click="$emit('update:modelValue', !modelValue)" round dense icon="menu" color="lp-primary"/>
+      <div v-if="$q.screen.gt.xs || !isSearchFieldActive" class="row justify-center items-center cursor-pointer" @click="$router.push({name: 'home'})">
+        <img v-if="$q.screen.sm" src="https://cdn.quasar.dev/logo-v2/svg/logo-dark.svg" width="48" height="48" alt="Quasar Logo">
+        <img v-else src="https://cdn.quasar.dev/logo-v2/svg/logo-horizontal-dark.svg" width="236" :height="$q.screen.xs? '24':'48'" alt="Quasar Logo">
+        <q-separator v-if="$q.screen.gt.xs" vertical color="lp-primary" class="q-ml-lg" />
```
Comment (n05la3) [inline docs/src/components/landing-page/MainLayoutHeader.vue:8]: This vertical bar is displayed only on sm viewports,

File: docs/src/layouts/MainLayout.vue (hunk 2 above)
Comment (IlCallo) [reply docs/src/layouts/MainLayout.vue]: If I'm not wrong, the whole footer toolbar is wrong in terms of content. We can probably use a better way to reuse the existing header and just make it change color scheme. Let's do this in a following PR, I'll just fix the color for now

**Pull request merged**